src/gui/application.cpp

```diff
@@ -232,6 +203,9 @@ Application::Application(int &argc, char **argv)
         }
     }
 
+    // needed during commandline options parsing
+    setApplicationVersion(_theme->versionSwitchOutput());
```

Comment:
The version is translatable but I guess we don't care...

Reply:
I considered data coming from the theme to be translated already...

Reply:
it is but if you  call setApplicationVersion before we set up the translator :D

Reply:
Will take this issue into account in my upcoming PR that fixes the translation of the commandline options.